src/node_options.h

```diff
@@ -199,6 +199,7 @@ class PerProcessOptions : public Options {
   std::string title;
   std::string trace_event_categories;
   std::string trace_event_file_pattern = "node_trace.${rotation}.log";
+  bool insecure_http = false;
```

Comment:
Can you move this to `PerEnvironmentOptions`?

Reply:
done